ci-operator/step-registry/ipi/conf/osimage/nutanix/ipi-conf-osimage-nutanix-ref.yaml

```diff
@@ -0,0 +1,18 @@
+ref:
+  as: ipi-conf-osimage-nutanix
+  from_image:
```

Comment:
```suggestion
  from: installer
```

Reply:
Could you tell when to use from: installer? I see a lot steps using from_image.

Reply:
Here you are using `openshift-install` to run the command to get rhcos image, different version of openshift installer print out different coreos boot image, so that means the step depends on the install payload, "from: installer" means the `installer` is from `release:latest` image which is configured in ci yaml file, while `from_image` is using a specific tag name of image, that means, whatever which ci job are triggered from, the step always the same installer from the fixed image.

Reply:
Looks like installer have not jq installed, what would you suggest? parse string?
`INFO[2024-05-20T10:21:41Z] Running step nutanix-ipi-proxy-fips-mini-perm-f28-ipi-conf-osimage-nutanix. 
INFO[2024-05-20T10:21:52Z] Logs for container test in pod nutanix-ipi-proxy-fips-mini-perm-f28-ipi-conf-osimage-nutanix: 
INFO[2024-05-20T10:21:52Z] Get default clusterOSImage
/bin/bash: line 11: jq: command not found`